subsys/bluetooth/shell/bt.c

```diff
@@ -697,6 +697,35 @@ static int cmd_name(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+static int cmd_appearance(const struct shell *sh, size_t argc, char *argv[])
+{
+	if (argc == 1) {
+		shell_print(sh, "Bluetooth Appearance: 0x%04x", bt_get_appearance());
+		return 0;
+	}
+
+	if (argc == 2) {
+		int err;
+		uint16_t new_appearance;
```

Comment:
This variable name is also super long. I think just `app` would be equally readable, since its use is constrained just over 10 lines of code or so (i.e. the context is clear to the reader).

Reply:
Ok. I'm accepting this under doubt. 'app' usually means 'application' after all.

Reply:
Will agree with @alwa-nordic that I wouldn't use `app` for appearance. What about something like `apprnc` or something like that? Personally I don't see the issue with long stack variable if they improve readability and understandability.